test_main.py

```diff
@@ -0,0 +1,61 @@
+import os
+import pytest
+
+from environment_variables import get_env
+from main import flickr_get_photos_list, flickr_get_photo
+
+
+def test_environment_variables_correct():
+    key = "TEST_VARIABLE"
+    value = "TEST_VALUE"
+    os.environ[key] = value
+    assert get_env(key) == value
+    os.environ.pop(key)
```

Comment:
If test fails, this variable will remain set in OS. Better to use fixture with cleanup (see pytest docs for fixtures)